projects.json

```diff
@@ -2525,5 +2525,44 @@
 	}
       }
     ]
+  },
+  {
+    "repository": "Git",
+    "url": "https://github.com/bizz84/SwiftyStoreKit",
+    "path": "SwiftyStoreKit",
```

Comment:
We like to keep the projects sorted alphabetically by 'path'. Would you mind putting the SwiftyStoreKit entry just after the SwifterSwift entry?